examples/exotica_examples/scripts/example_attach.py

```diff
@@ -40,7 +40,7 @@ def handleAttaching(i, x, scene):
 
 # Playback
 dt=0.03
-signal.signal(signal.SIGINT, sigIntHandler)
```

Comment:
This needs to stay - it's the only way to allow a graceful exit with Ctrl+C - otherwise you need to use ``kill` to kill Python.